docs/README.md

```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
+> ⚠️ _Both `imageSelector` and `phoneSelector` must be declared with valid values in order to provide Structured Data for SEO purposes._
 
 #### `store-name` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `text` | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
-| `tag` | `string` | HTML element to wrap the `store-name` block when rendered on the UI. | `div` |
+| Prop name |   Type   |                                                          Description                                                           | Default value |
+| :-------: | :------: | :----------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|  `text`   | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
+|   `tag`   | `string` |                              HTML element to wrap the `store-name` block when rendered on the UI.                              |     `div`     |
 
 #### `store-back-link` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
+| Prop name |   Type   |                          Description                          |    Default value     |
+| :-------: | :------: | :-----------------------------------------------------------: | :------------------: |
+|  `label`  | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
 
 #### `store-map` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `width` | `string` | Map width in pixels (`px`). | `100%` |
-| `height` | `string` | Map height in pixels (`px`). | `200px` |
+| Prop name |   Type   |         Description          | Default value |
+| :-------: | :------: | :--------------------------: | :-----------: |
+|  `width`  | `string` | Map width in pixels (`px`).  |    `100%`     |
+| `height`  | `string` | Map height in pixels (`px`). |    `200px`    |
 
 #### `store-address` props
 
-| Prop name | Type | Description | Default value     |
-| :-------: | :--: | :---------: | :---------------: |
-| `label` | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
+| Prop name |   Type   |                         Description                         |  Default value  |
+| :-------: | :------: | :---------------------------------------------------------: | :-------------: |
+|  `label`  | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
```

Comment:
No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!